fe/lib/main.dart

```diff
@@ -20,19 +24,302 @@ class MyApp extends StatelessWidget {
+      home: ContinuousScoreSheet(),
     );
   }
 }
 
-class ScoreSheet extends StatefulWidget {
-  const ScoreSheet({super.key});
+class ContinuousScoreSheet extends StatefulWidget {
+  const ContinuousScoreSheet({super.key});
+
+  @override
+  State<ContinuousScoreSheet> createState() => _ContinuousScoreSheetState();
+}
+
+class GroupInfo {
+  int startingMeasure;
+  double minY;
+
+  GroupInfo(this.startingMeasure, this.minY);
+}
+
+class MinMaxYCoords {
+  double minimumY;
+  double maximumY;
+
+  MinMaxYCoords(this.minimumY, this.maximumY);
+}
+
+class _ContinuousScoreSheetState extends State<ContinuousScoreSheet> {
+  final uri = 'http://localhost:3000'; // Replace this with localhost.run uri
+  late int _width;
+  late int _height;
+  late Future<SvgPicture> _svgs;
+  late Future<XmlDocument> _svgXml;
+  var groups = <GroupInfo>[];
+
+  // lineCoordinatesRegex captures the coordinates from
+  //  a string like "M171.66015625 189L472.4196980046949 189".
+  //  So for the above example, we capture the following:
+  //    x1 = 171.66015625, y1 = 189, x2 = 472.4196980046949, y2 = 189
+  //  Format: ?<x1> is an example of a named capture group
+  final RegExp lineCoordinatesRegex =
+      RegExp(r'M(?<x1>[\d\.]+) (?<y1>[\d\.]+)L(?<x2>[\d\.]+) (?<y2>[\d\.]+)$');
+
+  late final ScrollController _scrollController;
+
+  Future<http.Response> _getSvgLinks(int imageWidth) async {
+    final request =
+        http.MultipartRequest('POST', Uri.parse('$uri/musicxml-to-svg'));
+    request.fields['pageWidth'] = imageWidth.toString();
+
+    // TODO: Properly get the file instead of hardcoding a filename
+    const filename = "viva_la_vida.mxl";
+    final musicxmlBytes =
+        (await rootBundle.load('assets/$filename')).buffer.asUint8List();
+    request.files.add(http.MultipartFile.fromBytes('musicxml', musicxmlBytes,
+        filename: filename));
+
+    final streamResponse = await request.send();
+    final response = await http.Response.fromStream(streamResponse);
+
+    return response;
+  }
+
+  void _setupSvg() {
+    final response = _getSvgLinks(_width);
+    final svgDocument =
+        response.then((body) => XmlDocument.parse(utf8.decode(body.bodyBytes)));
+    final svgPicture =
+        response.then((body) => SvgPicture.memory(body.bodyBytes));
+
+    setState(() {
+      _svgs = svgPicture;
+    });
+    _svgXml = svgDocument;
+  }
+
+  void _setWidth() {
+    FlutterView view = WidgetsBinding.instance.platformDispatcher.views.first;
+    Size size = view.physicalSize / view.devicePixelRatio;
+    _width = size.width.toInt();
+    _height = size.height.toInt();
+  }
+
+  // Determine if elem is a 'measure' element e.g.
+  //  <g class="vf-measure" id="1">
+  bool _isMeasure(XmlElement elem) {
+    if (elem.getAttribute('class') == null) {
+      return false;
+    } else if (elem.getAttribute('class')!.contains('vf-measure') &&
+        elem.getAttribute('id') != '-1') {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
Just my personal opinion, something to make the code simpler
```suggestion
    final classAttribute = elem.getAttribute('class');
    return classAttribute != null 
      && classAttribute!.contains('vf-measure')
      && elem.getAttribute('id') != '-1';
```